schema/dataKeys.json

```diff
@@ -1601,17 +1652,80 @@
           "ootsign": {
```

Comment:
@HarelErlich I think that tests are failing because of missing `rpmPath` and `buildRpm` somewhere around here.